crates/wash-cli/tests/common/mod.rs

```diff
@@ -228,12 +229,27 @@ impl Drop for TestWashInstance {
 }
 
 /// Arguments for creating a new `TestWashInstance`
-#[derive(Debug, Default, PartialEq, Eq)]
+#[derive(Debug, PartialEq, Eq)]
 struct TestWashInstanceNewArgs {
     /// Extra arguments to feed to `wash up`
     pub extra_args: Vec<String>,
 }
 
+impl Default for TestWashInstanceNewArgs {
+    fn default() -> Self {
+        Self {
+            // we pin the optional wadm and wascloudhost versions
+            // to the current release, to avoid undefined behaviour in tests
+            extra_args: vec![
+                "--wadm-version".to_string(),
+                WADM_VERSION.to_string(),
+                "--wasmcloud-version".to_string(),
+                WASMCLOUD_HOST_VERSION.to_string(),
+            ],
+        }
+    }
+}
```

Comment:
I think this might cause some unintuitive behavior, but I do support adding this in our tests. I'm imagining writing a test in the future with something like
```rust
let mut args = TestWashInstanceNewArgs::default();
args.extra_args.extend(["--wadm-version", "v0.1.0"]);
```

and then being confused when my test failed because of duplicate args. Thoughts? I think it might be better to just not include this `impl Default`, but your create code is perfect because it will ensure that the arg is provided one way or another